perf/flame.jl

```diff
@@ -17,7 +17,8 @@ simulation = CA.get_simulation(config)
 
 import SciMLBase
 SciMLBase.step!(integrator) # compile first
-CA.call_all_callbacks!(integrator) # compile callbacks
+CA.call_all_callbacks!(integrator) # compile callbacks for out = ...
```

Comment:
I don't think you need to do this: the case with = should have been already called once when you get the integrator (https://github.com/CliMA/ClimaAtmos.jl/blob/2812637362192d774d14004a8a84e677dd21fd9a/src/solver/type_getters.jl#L891-L923).

Reply:
Ah, ok, I didn't realize we did that. Looks like that wasn't improving things anyway. I'm still not sure why there are more allocations when we provide more type info.